src/main/java/commands/DeleteCommand.java

```diff
@@ -0,0 +1,29 @@
+package commands;
+
+import DukeException.DukeException;
+import tasks.TaskList;
+import tasks.Task;
+
+public class DeleteCommand extends Command {
+    private final String details;
+
+    DeleteCommand(String details) {
+        this.details = details;
+    }
+    public void execute(TaskList tasks) throws DukeException {
```

Comment:
Perhaps it could be named executeTask() instead

Reply:
I'll work on it. Thanks!